pkg/virtctl/ssh/wrapped.go

```diff
@@ -0,0 +1,50 @@
+package ssh
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+func runLocalCommandClient(kind, namespace, name string) error {
+	args := []string{}
+	args = append(args, buildProxyCommandOption(kind, namespace, name))
+	args = append(args, buildSSHTarget(kind, namespace, name))
+
+	cmd := exec.Command("ssh", args...)
+	fmt.Println("running:", cmd)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	cmd.Stdin = os.Stdin
+
+	return cmd.Run()
+}
+
+func buildProxyCommandOption(kind, namespace, name string) string {
+	proxyCommand := strings.Builder{}
+	proxyCommand.WriteString("-o ProxyCommand=")
+	proxyCommand.WriteString(os.Args[0])
+	proxyCommand.WriteString(" portforward --forwardToStdout=true ")
+	proxyCommand.WriteString(fmt.Sprintf("%s/%s.%s", kind, name, namespace))
```

Comment:
Another nice side-effect of this is that the native and the wrapped client can easily find the same host key entries. :+1: